apps/heft/UPGRADING.md

```diff
@@ -1,5 +1,39 @@
 # Upgrade notes for @rushstack/heft
 
+### Heft 0.32.0
+
+This release of Heft removed the Jest plugin from the `@rushstack/heft` package
+and moved it to it's own package (`@rushstack/heft-jest-plugin`). To re-include
+Jest support in a project, include a dependency on `@rushstack/heft-jest-plugin`
+and add the following option to the project's `config/heft.json` file:
+
+```JSON
+{
+  "heftPlugins": [
+    {
+      "plugin": "@rushstack/heft-jest-plugin"
+    }
+  ]
+}
+```
+
+By default, configuration-relative module resolution will be performed for modules
+referenced in your Jest configuration. Existing "preset" configuration values will need
```

Comment:
I don't understand what this is saying. I asked @D4N14L about it offline. After he answers a few clarifying questions, I can propose a suggested rewording.

Reply:
I believe this should be addressed with the new text.